specification.scs

```diff
@@ -0,0 +1,22 @@
+ims_kb_specification
+    <- concept_reusable_component_specification;;
+
+ims_kb_specification = [*
+concept_ims_ostis_kb
```

Comment:
Seems like knowledge_base_IMS node as a start page and the root section of ims kb
```suggestion
knowledge_base_IMS
```
![image](https://user-images.githubusercontent.com/36566977/221361372-52da62d7-8b95-47d9-bc8d-dba7a5ab5100.png)